db/schema.rb

```diff
@@ -274,15 +274,15 @@
     t.string "reference", null: false
     t.string "move_type"
     t.string "additional_information"
-    t.integer "nomis_event_id"
```

Comment:
Weird but these haven't changed?

Reply:
@willfish sorry, I shouldn't have committed that: just annoying jumping fields because they are ordered a bit differently in different local databases. Now fixed.

Reply:
No worries :)